packages/ui/src/views/UserSettings.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <MainLayout>
+    <div class="container">
+      <SectionHeader title="SETTINGS" />
+      <form class="form">
+        <CustomInput
+          data-lpignore="false"
+          type="text"
+          label="name"
+          :value="player.socials?.name"
+          @change="setValue"
+        />
+        <CustomInput
+          data-lpignore="false"
+          type="text"
+          label="company"
+          :value="player.socials?.company"
+          @change="setValue"
+        />
+        <CustomInput
+          data-lpignore="false"
+          type="text"
+          label="twitter"
+          :value="player.socials?.twitter"
+          @change="setValue"
+        />
+        <CustomInput
+          data-lpignore="false"
+          type="text"
+          label="discord"
+          :value="player.socials?.discord"
+          @change="setValue"
+        />
+        <CustomInput
+          data-lpignore="false"
+          type="text"
+          label="telegram"
+          :value="player.socials?.telegram"
+          @change="setValue"
+        />
```

Comment:
Could we create an `v-for` for all of those `CustomInput`s?

Reply:
This will be implemented in #10